resources/assets/js/app/App.js

```diff
@@ -81,6 +83,7 @@ class App extends React.Component {
             : <div>
                 {this.props.children}
                 <Notifications userId={AuthService.getUserId()} />
+                <GeneralChannel isAuthorized={AuthService.isAuthorized()} />
```

Comment:
you can attach to this component appropriate global state instead of pass params there